kmapper/kmapper.py

```diff
@@ -25,6 +25,7 @@
     graph_data_distribution,
     colorscale_default,
 )
+from scipy.sparse import hstack, csr
```

Comment:
Could you please combine this import with the other `scipy.sparse` imports. It should read
```
from scipy.sparse import issparse, hstack, csr
```